submissions/DmytryjK/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+'user strict';
+//content
+const contentArray = [
```

Comment:
Don’t use names as arr, array, data or etc. They to generic. contentArray also a bad idea - you show data type in the variable name and in JS it’s redundant

Reply:
Fixed, thx)